### ISSUE TITLE
feat: `onReaction`

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2629]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.33.3](https://github.com/open-wa/wa-automate-nodejs/compare/4.33.2...4.33.3) (2022-04-07)

- Release 4.33.3 [`087ad9c`](https://github.com/open-wa/wa-automate-nodejs/commit/087ad9c465d42ad07cebd62b3c73c389d1751aa7)
- ✨ feat: [INSIDERS] `Client.onReaction` #2629 [`cb000bb`](https://github.com/open-wa/wa-automate-nodejs/commit/cb000bb2b35503891bb74863383418548052782f)
- 🥚 patches update [`04f6a74`](https://github.com/open-wa/wa-automate-nodejs/commit/04f6a7432dbdd3d49a34d96e189faf3f14faf36e)
- 🥚 patches update [`d486060`](https://github.com/open-wa/wa-automate-nodejs/commit/d486060c0b40d6a25d856c252b113b815adf9343)
- updated types-only package [`1707f09`](https://github.com/open-wa/wa-automate-nodejs/commit/1707f09928b4140e5ba3decb1216e97d95775da6)

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.33.4](https://github.com/open-wa/wa-automate-nodejs/compare/4.33.3...4.33.4) (2022-04-07)

- Release 4.33.4 [`6bd6d5c`](https://github.com/open-wa/wa-automate-nodejs/commit/6bd6d5ced954db6e1959b839cc2cb402b8656611)
- 📝 fix reaction docs [`747359f`](https://github.com/open-wa/wa-automate-nodejs/commit/747359f81b9a4d1f0bd723232e778f846943c6c2)
- updated types-only package [`32d5e72`](https://github.com/open-wa/wa-automate-nodejs/commit/32d5e72431df1b238b3ae327422192e9cff7118a)
- 🏷️ export reaction types #2629 [`58bccfd`](https://github.com/open-wa/wa-automate-nodejs/commit/58bccfdccaef1248e53d73fc40661d65fbe5724f)
